pkgs/development/compilers/emscripten/default.nix

```
@@ -38,6 +39,10 @@ stdenv.mkDerivation rec {
       src = ./0001-emulate-clang-sysroot-include-logic.patch;
       resourceDir = "${llvmEnv}/lib/clang/${llvmPackages.release_version}/";
     })
+    (fetchpatch {
+      url = "https://github.com/emscripten-core/emscripten/pull/16986.patch";
+      sha256 = "0wbnvdy6gyizmrvqwbdz9k2cs9cmni49iqn1hggsfcnkwn2qlddp";
```

```suggestion
      # https://github.com/emscripten-core/emscripten/pull/16986
      url = "https://github.com/emscripten-core/emscripten/commit/d5ef6937fe395488e23a82c1e582a7ea5c2dab83.patch";
      sha256 = "sha256-YX5DG8i5x6S7XnU58etEapDd+o5SuzbFIGv8v/9+T3E=";
```
Might be cleaner to reference the actual commit that got merged upstream (also, the `sha` on the PR's patch was causing an error for me -- I had to run nix-build twice to get a working one), and then note the PR via a comment.